.github/workflows/cherry-pick.yml

```diff
@@ -39,4 +39,54 @@ jobs:
           git cherry-pick -x ${{ github.event.inputs.shas }}
           git push --set-upstream https://github.com/ampproject/amphtml.git cherry-pick-${{ github.event.inputs.amp-version }}
 
-      # TODO(estherkim): tie to promote workflow
+  get-channels-to-promote:
```

Comment:
Also let's add a checkbox to the workflow_dispatch event that lets the on-duty choose whether to run the promote workflow or not (defaults to yes, like in the Google dashboard)